libs/datasets/sources/covid_tracking_source.py

```diff
@@ -100,7 +101,7 @@ def standardize_data(cls, data: pd.DataFrame) -> pd.DataFrame:
         data[cls.Fields.AGGREGATE_LEVEL] = AggregationLevel.STATE.value
         # Date checked is the time that the data is actually updated.
         # assigning the date field as the date floor of that day.
-        data[cls.Fields.DATE] = data[cls.Fields.DATE_CHECKED].dt.floor("D")
+        data[cls.Fields.DATE] = data[cls.Fields.DATE_CHECKED].dt.tz_localize(None).dt.floor("D")
```

Comment:
`tz_localize(None)` seems to just strip the time in a way that removes timezone so that the time is presumed local (for any definition of local).

What do you think about using `.normalize()` (possibly `.normalize().date()`)  instead of stripping localization and doing a `floor("D")`?

Reply:
I like the idea of it, but we actually have to have this be non timezoned times (none of the other data sources have time zones, so this having a timezone actually breaks comparison with dates from other data sources in some operations).